tools/Jenkinsfile-PR

```diff
@@ -0,0 +1,24 @@
+pipeline {
+	environment {
+		ANDROID_HOME = '/opt/imint/android-sdk'
+		ANDROID_NDK_HOME = '/home/jenkins/android-ndk-r15c'
+		DISPLAY = ':0'
+	}
+	agent none // Force stages to specify agent
+	stages {
+		stage('parallel') {
+			failFast false
+			parallel {
```

Comment:
maybe I can remove the "parallell" stuff too

Reply:
Yes remove it :+1: